cmd/ignite/run/create.go

```diff
@@ -95,6 +97,9 @@ func (cf *CreateFlags) NewCreateOptions(args []string, fs *flag.FlagSet) (*Creat
 		return nil, err
 	}
 
+	// Save env variables
+	baseVM.SetAnnotation(constants.IGNITE_ENV_VAR_ANNOTATION, cf.EnvVars)
```

Comment:
I think we need to provide a way to set this `cf.EnvVars` via the ignite configuration (global) as well, like other similar VM configurations in https://github.com/weaveworks/ignite/blob/999ad7c5ed35b8677a3950a0c227e78ee7d0956e/pkg/apis/ignite/types.go#L261-L266 to allow setting default for all the VMs and allow overriding that through `cf` flag and individual VM configuration.
Since this is set as annotation, the baseVM should have the annotation inherited from the ignite configuration. Annotation in VM configuration should override/append the baseVM configuration. The VM create flag value should append/override everything before.